### DISCUSSION TITLE
Filter by tile name

### DISCUSSION THREAD

**Discussion by Squiccio** [CS-SI/eodag#671] [Q&A] [answered]
Is it possible to make a search by tilename for sentinel 2 data... i tried but nothing worked . 
example, to search for T32TPT tiles , i tried this: eodag search --productType S2_MSI_L2A --start 2023-03-01 --query Name=T32TPT
thanks

**Comment by sbrunato** [comment]
Hello @Squiccio , this is not standardized yet (see https://github.com/CS-SI/eodag/issues/255), but for the moment you can use:
- for `peps`: 
```sh
eodag search --productType S2_MSI_L1C --start 2023-03-01 --query tileid=32TPT
```
- for `mundi`, `onda`: 
```sh
eodag search --productType S2_MSI_L1C --start 2023-03-01 --query tileIdentifier=32TPT
```
- for `creodias`, `cop_dataspace`: 
```sh
eodag search --productType S2_MSI_L1C --start 2023-03-01 --query tileId=32TPT
```
- for `earth_search`:
```sh
eodag search --productType S2_MSI_L2A --start 2023-03-01 --query "sentinel:utm_zone=32&sentinel:latitude_band=T&sentinel:grid_square=PT"
```
- for `theia`: 
```sh
eodag search --productType S2_MSI_L2A_MAJA --start 2023-03-01 --query location=T32TPT
```

**Reply by Squiccio** [reply]
Thanks it works on cop_dataspace

**Reply by Squiccio** [reply]
Hello @sbrunato i made a search like you suggested but it works only for dates in past months. If i set the start date to february, i find 6 products (eodag search --productType S2_MSI_L1C --start 2023-02-01 --query tileId=32TPT) but if i set a date from begenning of march, nothing is found and this is not possible because i found some products on this tile in thecop dataspace browser.
so i  don't know where is the problem.
Thanks for your help

**Reply by sbrunato** [reply]
I also noticed a similar issue, and submitted it to Copernius Dataspace / Creodias. Until this is fixed, you may try to execute your request on another provider

**Reply by sbrunato** [reply, marked as answer]
Hello @Squiccio , this has just been resolved with https://github.com/CS-SI/eodag/pull/713 and will be available in incoming `v2.10.0`
